### PR TITLE
Fix CS bis

### PR DESCRIPTION
**Pull request by deguif**
See https://github.com/ruflin/Elastica/pull/1695#discussion_r347510004 for the new rule

### CODE REVIEW

**Comment by deguif** [comment]
Ok, let's close it then. I dumped the query result and this is not working.

File: lib/Elastica/Aggregation/Derivative.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function setBucketsPath(string $bucketsPath)
      *
      * @return $this
```
Comment (thePanz) [inline lib/Elastica/Aggregation/Derivative.php]: The `return $this` should be removed too :)

File: lib/Elastica/Transport/HttpAdapter.php [hunk 2]
```diff
@@ -32,7 +32,7 @@ class HttpAdapter extends AbstractTransport
      *
      * @param Connection $connection
      */
-    public function __construct(Connection $connection = null, HttpAdapterInterface $httpAdapter)
+    public function __construct(?Connection $connection = null, HttpAdapterInterface $httpAdapter)
```
Comment (deguif) [inline lib/Elastica/Transport/HttpAdapter.php]: There is a glitch here, optional argument before a non optional argument.

**Comment by ruflin** [comment]
@deguif Any chance you could update the PR description to contain a bit more details on the change? I'm aware the comment is linked but if someone comes back to this and asking why this all changed, would be nice to have it in a single place summarised.

File: lib/Elastica/Aggregation/Derivative.php (hunk 1 above)
Comment (ruflin) [reply lib/Elastica/Aggregation/Derivative.php]: Lets separate PR's which are fully automated and manual changes ;-)

File: lib/Elastica/Transport/HttpAdapter.php (hunk 2 above)
Comment (ruflin) [reply lib/Elastica/Transport/HttpAdapter.php]: If we don't make it optional, would things break?

File: .php_cs.dist [hunk 3]
```diff
@@ -19,6 +19,8 @@ $config = PhpCsFixer\Config::create()
         '@PHPUnit60Migration:risky' => true,
         'php_unit_dedicate_assert' => ['target' => 'newest'],
         'native_function_invocation' => true,
+        'no_alias_functions' => true,
```
Comment (thePanz) [inline .php_cs.dist]: What about adding `@PhpCsFixer` and `@PHP71Migration` too?

Comment (ruflin) [reply .php_cs.dist]: No clue what the effect will be but I suggest we do this in a follow up PR to keep PRs small.

Comment (ruflin) [reply .php_cs.dist]: MR = PR? So you are ok to merge and then follow up? :-)

**Comment by thePanz** [comment]
@ruflin Ok to merge! I'd squash the changes, just to keep the git history clean. wdyt?

**Comment by ruflin** [comment]
@thePanz Yes, I always squash. Commit message is now not optimal because I took just the PR one but well ... @deguif Could you still update the PR description?

**Comment by deguif** [comment]
@ruflin @thePanz Yes, will do that

**Comment by deguif** [comment]
@ruflin @thePanz description should now better reflect the changes introduced here.

**Pull request merged**